skll/learner/__init__.py

```diff
@@ -425,19 +442,20 @@ def _convert_coef_array_to_feature_names(self, coef, feature_name_prefix=""):
 
         Parameters
         ----------
-        coef : np.array
+        coef : numpy.ndarray
             A numpy array with the model coefficients
-        feature_name_prefix : str, optional
+        feature_name_prefix : str
             An optional string that should be prefixed to the feature
             name, e.g. the name of the class for LogisticRegression
             or the class pair for SVCs with linear kernels.
 
         Returns
         -------
-        res : dict
+        Dict[str, Any]
             A dictionary of labeled weights
```

Comment:
`Any` feels too broad given the description of "A dictionary of labeled weights".  Only commenting here, but this applies to all uses of `Dict[str, Any]`.

Reply:
Yes, some dictionary structures are too broad to properly provide type hints for. If I use those very specific type hints, then mypy requires a lot more checks. So, this seems like a reasonable enough cop-out :)